docs/content/getting-started.mdx

```diff
@@ -1,145 +1,141 @@
+---
+title: "Welcome to Dagster! | Dagster Docs"
+---
+
 # Welcome to Dagster!
 
 <p className="text-2xl mt-0 text-gray-500 tracking-tight font-light">
   Dagster is the data orchestration platform built for productivity.
 </p>
 
-Get started with Dagster in just three quick steps:
-
-1. [Install Dagster](#step-1-install-dagster)
-2. [Define assets](#step-2-define-assets)
-3. [Materialize the assets](#step-3-materialize-the-assets)
-
----
+Dagster is a system for building and maintaining data assets, such as tables, data sets, machine learning models, or reports.
 
-## Step 1: Install Dagster
+With Dagster, you declare the assets that you want to exist, the dependencies between them, and how they should be updated. Dagster then takes responsibility for keeping your assets up-to-date, as well as for tracking their history.
 
-<Note>
-  Dagster requires Python 3.6+. Refer to the{" "}
-  <a href="/getting-started/install">Installation documentation</a> for more
-  info.
-</Note>
-
-To install Dagster into an existing Python environment, run:
-
-```bash
-pip install dagster
-```
-
-This installs the latest stable version of the core Dagster packages in your current Python environment.
+<div className="inline-flex flex-row space-x-4">
+    <button className="py-1 px-4 rounded-lg bg-gable-green text-white hover:bg-transparent hover:text-gable-green hover: border hover:border-gable-green">
+        Set up Dagster
```

Comment:
My impulse would be to leave "Set up Dagster" out. I think there's high cost to every choice that we make the reader make, and I think few people who are new to Dagster are going to want to immediately start just installing it - what would they do after that?  If we do want to include guidance here on how to install Dagster, I think it would be better to include what we have on the current "Getting Started" page - i.e. the "pip install", which immediately shows off how simple it is to install Dagster, and avoids the reader needing to make a choice.